static/js/src/app/gophish.js

```diff
@@ -223,6 +223,28 @@ var api = {
             return query("/users/" + id, "DELETE", {}, true)
         }
     },
+    webhooks: {
+        get: function() {
+            return query("/webhooks/", "GET", {}, false)
+        },
+        post: function(webhook) {
+            return query("/webhooks/", "POST", webhook, false)
+        },
+    },
+    webhookId: {
+        get: function(id) {
+            return query(`/webhooks/${id}`, "GET", {}, false)
```

Comment:
This particular file isn't currently built with Babel compilation, so we shouldn't use ES6 directives here. If you could, please change these to use the same double-quote string styles as the other endpoints.

The thing to check is whether or not running `gulp` works as expected.

Reply:
will change

Reply:
fixed. 

I need to install gulp to check that

Reply:
running `gulp` has worked well.

Reply:
```
$ gulp
[10:04:42] Using gulpfile ~/projects/go_lang/gophish/gulpfile.js
[10:04:42] Starting 'default'...
[10:04:42] Starting 'vendorjs'...
[10:04:42] Starting 'scripts'...
[10:04:42] Starting 'styles'...
[10:04:44] Finished 'scripts' after 1.32 s
[10:04:53] Finished 'styles' after 10 s
[10:04:53] Finished 'vendorjs' after 10 s
[10:04:53] Finished 'default' after 10 s
```